bin/megafusion.py

```diff
@@ -0,0 +1,196 @@
+#!/usr/bin/env python
```

Comment:
Can we have more info about where this script is coming from in comments around here?

Reply:
+ Licensing would be amazing

Reply:
I agree the script is not yet optimal, function definition and comments have to be added. It is meant as the first version. More functionalities and better documentation will be added (issue https://github.com/nf-core/rnafusion/issues/356)